Source/SwiftLintBuiltInRules/Rules/Idiomatic/MultipleStatementsDeclarationRule.swift

```diff
@@ -0,0 +1,92 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule(explicitRewriter: true, optIn: true)
+struct MultipleStatementsDeclarationRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "multiple_statements_declaration",
+        name: "Multiple Statements Declaration",
+        description: "Statements should not be on the same line",
+        kind: .idiomatic,
+        nonTriggeringExamples: [
+            Example("let a = 1;\nlet b = 2;"),
```

Comment:
Please use multi-line string blocks instead of explicit newline characters `\n`.